### ISSUE TITLE
Live TV momentary judder

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#46]
```
What steps will reproduce the problem?
1. Watch live TV 
2.
3.

What is the expected output? What do you see instead?
Video shouldn't judder

What version of the product are you using? On what operating system?
SVN1260 on XBMC Live (Linux Jaunty)

Please provide any additional information below.
The live channel feed judders every 2-3seconds (hardly noticeable) this 
problem isn't a backend issue as when I use the XBMC native client Live TV 
plays without any issues

```

Original issue reported on code.google.com by `richard....@gmail.com` on 6 Aug 2009 at 7:29

**Comment by GoogleCodeExporter** [comment]
```
I'm not the developer and cannot really help you, but I think it would help if 
you 
could provide logs and screenshots the issue might be easier to resolve.

Regards, Pär
```

Original comment by `par.tojkander@gmail.com` on 24 Aug 2009 at 12:59

**Comment by GoogleCodeExporter** [comment]
```
Would need a log file to be able to chime in on this one.
```

Original comment by `semir.pa...@gmail.com` on 28 Aug 2009 at 10:00
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Sorry for the delay folks; here's the log files which should be 'clean' and 
show a 
period of MythBox running for about 20 seconds during which time 3 judders 
occur.

XBMC LOG: http://pastebin.com/m6220670b

MYTHBOX LOG: http://pastebin.com/m6aca6217

The Recordings & LiveTV Directories reside on a separate server and the paths 
are 
mounted on my HTPC using NFS.

```

Original comment by `richard....@gmail.com` on 29 Aug 2009 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
Please retest on latest codebase. Install mercurial and run from your scripts 
dir:

hg clone http://mythbox.googlecode.com/hg MythBox
```

Original comment by `semir.pa...@gmail.com` on 19 Oct 2009 at 6:18

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svn 1510 for 0.21 backends.
```

Original comment by `semir.pa...@gmail.com` on 21 Oct 2009 at 1:18
- Changed state: **Fixed**
